### DISCUSSION TITLE
Getting an error while building

### DISCUSSION THREAD

**Discussion by AkshaySC1188** [lencx/ChatGPT#180] [Q&A] [answered]
I am trying to build the application after downloading the .zip just to learn the process but i am getting an error. First of all, I am a complete noob when it comes to programing and this is the first time i am building. I installed Rust, VS Code and Tauri just like its mentioned on the page before building.

I am getting the following error.

error: proc macro panicked
  --> src\main.rs:24:19
   |
24 |     let context = tauri::generate_context!();
   |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: message: The `distDir` configuration is set to `"../dist"` but this path doesn't exist

Does anyone know why that can be?

I apologies if i am bothering you guys but any help is appreciated.

**Comment by lencx** [comment, marked as answer]
You can create an empty folder named `dist` under the root path.

**Reply by AkshaySC1188** [reply]
Got it working...Thank you for your prompt response and keep up the good work you are amazing.
